packages/flash-swap/test/integration/collateralFlashUniswapV2/CollateralFlashUniswapV2.ts

```diff
@@ -1,35 +1,37 @@
 import { integrationFixture } from "../../shared/fixtures";
-import { shouldBehaveLikeHifiFlashUniswapV2 } from "./HifiFlashUniswapV2.behavior";
+import { shouldBehaveLikeCollateralFlashUniswapV2 } from "./CollateralFlashUniswapV2.behavior";
 
-export function integrationTestHifiFlashUniswapV2(): void {
-  describe("HifiFlashUniswapV2", function () {
+export function integrationTestCollateralFlashUniswapV2(): void {
+  describe("CollateralFlashUniswapV2", function () {
     beforeEach(async function () {
       const {
         balanceSheet,
         fintroller,
         hToken,
-        hifiFlashUniswapV2,
+        collateralFlashUniswapV2,
         maliciousPair,
         oracle,
+        underlyingFlashUniswapV2,
+        uniswapV2Pair,
         usdc,
         usdcPriceFeed,
-        uniswapV2Pair,
         wbtc,
         wbtcPriceFeed,
       } = await this.loadFixture(integrationFixture);
       this.contracts.balanceSheet = balanceSheet;
       this.contracts.fintroller = fintroller;
       this.contracts.hToken = hToken;
-      this.contracts.hifiFlashUniswapV2 = hifiFlashUniswapV2;
+      this.contracts.collateralFlashUniswapV2 = collateralFlashUniswapV2;
```

Comment:
@paulrberg would be great to re-order alphabetically and place this line right below the balanceSheet contract assignment.